tools/pythonpkg/src/python_udf.cpp

```diff
@@ -181,43 +208,48 @@ static scalar_function_t CreateVectorizedFunction(PyObject *function, PythonExce
 				input.Slice(selvec, index);
 			}
 		}
+		idx_t count;
```

Comment:
Can you explain what changes we're making below? The diff is quite hard to read and I can't make out what is actually different